src/templates/layouts/event/util.tsx

```diff
@@ -0,0 +1,72 @@
+import _ from 'lodash'
```

Comment:
i would move all of these date time utils to our existing `lib/utils/date.ts` file.